config/development/machineclasses.json

```diff
@@ -1,20 +1,20 @@
 [
   {
-    "name": "extra-huge",
+    "name": "libvirt-provider-dev-extra-huge",
```

Comment:
why? was it problem?

Reply:
Yes, according to the current configuration, applying the customization results in all resources being created and the DaemonSet coming to a running state. However, when querying the machine pool, the machine classes are blank. This issue prevents the machine from being reconciled in the pod.